app/controllers/v0/feedbacks_controller.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+module V0
+  class FeedbacksController < ApplicationController
+    include SentryLogging
+    include ActionController::ParamsWrapper
+    wrap_parameters Feedback, format: :json
```

Comment:
ya, it looks like it was meant for the entirety of `vets-api`
https://github.com/department-of-veterans-affairs/vets-api/blob/master/config/initializers/wrap_parameters.rb

but it doesnt work as written.